footer.php

```diff
@@ -13,6 +13,7 @@
 		</div><!-- .container -->
 	</div><!-- #content -->
 
+	<div class="clear"></div>
```

Comment:
We should also get rid of this one too - https://github.com/siteorigin/siteorigin-unwind/blob/develop/inc/template-tags.php#L168